### PR TITLE
Feat/recovy account

### PR DESCRIPTION
**Pull request by 3fili3**
# Recover and confirm email 

## Description

This is the logic to make a user to recover their account and confirm their email.

List any external dependencies that are required for this change.

- nodemailer

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Linting and other (non-breaking change which adds spacing or other small lint fixes)
- Performance (non-breaking change which refactors the code to be more performant )

## How Has This Been Tested?

- Describe any manual testing done to verify your changes.
  - Filiberto and I went over the changes and made sure that no existing tests were broken.
- List any relevant details for your test configuration as well an explanation.
  - <ANSWER_HERE>

## Checklist

- [x] I have performed a self-review of my own code
  - [x] I have checked my code and corrected any misspellings
  - [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if necessary)
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by 3fili3** [comment]
Confirm account test

File: src/services/nodemailer.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        const transporter = nodemailer.createTransport({
+            host   : hostSupport,
+            port   : portSupport,
+            secure : secureSupport,
+            auth
+        });
+
+        const send = (to = "", title = "", template = "") => {
+            transporter.sendMail({
+                from    : auth.user, to,
```
Comment (pancho508) [inline src/services/nodemailer.js:22]: Separate into separate lines

File: src/modules/users/users.ctrl.js [hunk 2]
```diff
@@ -143,6 +142,71 @@ const deleteRel = async ({ services }, body) => {
...
+// Other User Controllers
+const WaitingForAccountConfirmation = async ({ services }, body) => {
+  MapConfirmAccount.set(body.email, body);
+
+  const token = jwt.createToken({ email: body.email });
+
+  services.email.send(
+    body.email,
+    "Confirmación de Cuenta",
+    services.template.confirmEmail(body.userName, "http://localhost:5173/cuenta/" + token + "/confirmacion")
```
Comment (pancho508) [inline src/modules/users/users.ctrl.js:154]: Change it to dynamic

File: src/modules/users/users.ctrl.js [hunk 3]
```diff
@@ -143,6 +142,71 @@ const deleteRel = async ({ services }, body) => {
...
+  );
+
+  return { data: "Se ha enviado un mensaje a " + body.email + " para confirmar tu cuenta" };
+};
+
+const recoveryAccount = async ({ services }, body) => {
+  let data = await services.neo4j.session.run(logInQuery(body));
```
Comment (pancho508) [inline src/modules/users/users.ctrl.js:161]: This is always going to throw an error, almost all the queries filter out the deleted accounts

**Review by pancho508 — APPROVED** [review]

**Pull request merged**